modules/network-observability-virtualization-configuration.adoc

```diff
@@ -0,0 +1,88 @@
+
+.Procedure
+. Get information about the virtual machine launcher pod by running the following command. This information is used in Step 5: 
++
+[source,terminal]
+----
+$ oc get pod virt-launcher-<vm_name>-<suffix> -n <namespace> -o yaml
+----
++
+[source,yaml]
+----
+$ oc get pod virt-launcher-fedora-aqua-fowl-13-zr2x9 -n my-vms -o yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/network-status: |- 
+      [{
+        "name": "ovn-kubernetes",
+        "interface": "eth0",
+        "ips": [
+          "10.129.2.39"
+        ],
+        "mac": "0a:58:0a:81:02:27",
+        "default": true,
+        "dns": {}
+      },
+      {
+        "name": "my-vms/l2-network",   <1>
+        "interface": "podc0f69e19ba2", <2>
+        "ips": [
+          "10.10.10.15"
+        ], <3>
+        "mac": "02:fb:f8:00:00:12",    <4>
+        "dns": {}
+      }]
+  name: virt-launcher-fedora-aqua-fowl-13-zr2x9
+  namespace: my-vms
+spec:
+  ...
+status:
+  ...
+----
+<1> The name of the secondary network.
+<2> The network interface name of the secondary network.
+<3> The list of IPs used by the secondary network.
+<4> The MAC address used for secondary network.
+
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
+. Select *cluster* and then select the *YAML* tab.
+. Configure `FlowCollector` based on the information you found from the additional network investigation. 
++
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+# ...
+ebpf:
+    privileged: true              <1>
+  processor:
+    advanced:
+      secondaryNetworks:
+      - index:                    <2>
+        - MAC                     <3>
+        name: my-vms/l2-network   <4> 
+# ...
+----
+<.> Ensure that the eBPF agent is in `privileged` mode so that flows are collected for secondary interfaces.
+<.> Define the fields to use for indexing the virtual machine launcher pods. It is recommended to use the `MAC` address as the indexing field to get network flows enrichment for secondary interfaces. If you have overlapping MAC address between pods, then additional indexing fields, such as `IP` and `Interface`, could be added to have accurate enrichment.
+<.> If your additional network information has a MAC address, add `MAC` to the field list.
+<.> Specify the name of the network found in `k8s.v1.cni.cncf.io/network-status` annotation. Usually <namespace>/<network_attachement_definition_name>. 
+
+. Observe VM traffic:
+.. Navigate to the *Network Traffic* page.
+.. Filter by *Source* IP using your virtual machine IP found in `k8s.v1.cni.cncf.io/network-status` annotation.
+.. View both *Source* and *Destination* fields should enriched identifying VM launcher Pods and VM instance as Owners
```

Comment:
I've re-read this sentence a couple times and still can't quite figure out what it's trying to say. I think it's missing a few words. I can't even give a suggested rewrite. I think you at least need a "the" in front of the field names. Is it trying to say to check these two fields to ensure that <something with the end of the sentence>.

Also needs a period at the end, and do "Pods" and "Owners" have to be capitalized?

Reply:
yeah I think its missing some words. It should be 
> View both *Source* and *Destination* fields, which should be enriched, and identify the VM launcher pods and the VM instance as owners.